{"name": "test_invalid_sign_in_get_request", "status": "passed", "steps": [{"name": "Fail sign in get request", "status": "passed", "start": 1648789109840, "stop": 1648789110042}], "start": 1648789109839, "stop": 1648789110042, "uuid": "109633f9-d4ed-4c8f-90ca-ddfdcede5370", "historyId": "b76b91d68b162f208ab544833704df5e", "testCaseId": "c03d7c083b451bb970c52430f0e4a590", "fullName": "tests.api_testcases.login_test.login_test.TestUserLogin#test_invalid_sign_in_get_request", "labels": [{"name": "parentSuite", "value": "tests.api_testcases.login_test"}, {"name": "suite", "value": "login_test"}, {"name": "subSuite", "value": "TestUserLogin"}, {"name": "host", "value": "pop-os"}, {"name": "thread", "value": "81456-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.api_testcases.login_test.login_test"}]}